packages/react-components/react-switch/src/components/Switch/useSwitchStyles.ts

```diff
@@ -126,21 +126,21 @@ const useInputStyles = makeStyles({
     // Enabled and checked
     ':enabled:checked': {
       [`& ~ .${switchClassNames.indicator}`]: {
-        backgroundColor: tokens.colorBrandBackground,
-        color: tokens.colorNeutralForegroundOnBrand,
+        backgroundColor: tokens.colorCompoundBrandBackground,
```

Comment:
Is the figma design spec updated to reference the compound tokens?

Reply:
Yes it is